terkin/sensor/core.py

```diff
@@ -275,7 +277,13 @@ def start(self):
         # Todo: Improve error handling.
         try:
             from onewire import OneWire
-            self.adapter = OneWire(Pin(self.pins['data']))
+            if platform_info.vendor == platform_info.MICROPYTHON.Vanilla:
+                self.adapter = OneWire(Pin(int(self.pins['data'][1:])))
+            elif platform_info.vendor == platform_info.MICROPYTHON.Pycom:
+                self.adapter = OneWire(Pin(self.pins['data']))
+            else:
+                raise NotImplementedError('I2C Bus is '
+                        'not implemented on this platform')
```

Comment:
The changes from 7330d65a might already account for that and even will handle the collision between both drivers. Remember to invoke `make setup` again to get both drivers into `dist-packages` as `onewire_native.py` vs. `onewire_python.py`.